BepInEx.IL2CPP/ProxyAssemblyGenerator.cs

```diff
@@ -42,19 +43,34 @@ public static AppDomain CreateDomain(string name, AppDomainSetup setup)
 
 		private static string HashPath => Path.Combine(Preloader.IL2CPPUnhollowedPath, "assembly-hash.txt");
 
+		private static string UnityBaseLibsDirectory => Path.Combine(Paths.BepInExRootPath, "unity-libs");
+
 		private static string tempDumperDirectory => Path.Combine(Preloader.IL2CPPUnhollowedPath, "temp");
 
 		private static ManualLogSource Il2cppDumperLogger = Logger.CreateLogSource("Il2CppDumper");
 
-
-		private static string ComputeGameAssemblyHash()
+		private static string ComputeHash()
 		{
 			using var md5 = MD5.Create();
-			using var assemblyStream = File.OpenRead(GameAssemblyPath);
 
-			var hash = md5.ComputeHash(assemblyStream);
+			var gameAssemblyBytes = File.ReadAllBytes(GameAssemblyPath);
+			md5.TransformBlock(gameAssemblyBytes, 0, gameAssemblyBytes.Length, gameAssemblyBytes, 0);
+
+			if (Directory.Exists(UnityBaseLibsDirectory))
+			{
+				foreach (var file in Directory.EnumerateFiles(UnityBaseLibsDirectory, "*.dll", SearchOption.TopDirectoryOnly))
+				{
+					var pathBytes = Encoding.UTF8.GetBytes(Path.GetFileName(file));
+					md5.TransformBlock(pathBytes, 0, pathBytes.Length, pathBytes, 0);
 
-			return Utility.ByteArrayToString(hash);
+					var contentBytes = File.ReadAllBytes(file);
+					md5.TransformBlock(contentBytes, 0, contentBytes.Length, contentBytes, 0);
+				}
```

Comment:
Did you measure what the inpact on load times with this? Or at least how long this takes on average?